src/mumble/mumble.pro

```diff
@@ -11,6 +11,8 @@ DEFINES *= MUMBLE
 TEMPLATE = app
 TARGET = mumble
 
+QT += concurrent
```

Comment:
should use `*=`, also, there are already a bunch of these L66, maybe add it there.

Reply:
Yeah. See #4053